rebuild-docs

```diff
@@ -12,6 +12,13 @@ rm -rf ${core_dir} && git clone https://github.com/canonicalltd/ubuntu-core-docs
 rm -rf ${phone_dir} && git clone https://github.com/ubuntudesign/phone-docs.git ${phone_dir} || true
 #git clone https://github.com/canonicalltd/snapcraft-docs.git ${snapcraft_dir}
 
+# Extra build steps for Ubuntu Core docs
+(
+  cd ${core_dir}
+  repo init -u git@github.com:CanonicalLtd/ubuntu-core-docs.git
```

Comment:
We need to add some info on installing the repo command to the repository like there is in the ubuntu-core-docs repository.
(I'm slightly sad there seems to be no easier way of installing it.)